src/chrome/content/rules/laurette-theatre.fr.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="laurette-theatre.fr">

```

Comment:
There are various subdomains like `[language code].laurette-theatre.fr` that serving a mismatching cert. Could you add a hint about this misbehavior?